src/System.Private.CoreLib/src/System/DateTime.Unix.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    public readonly partial struct DateTime
+    {
+        internal static readonly bool s_isLeapSecondsSupportedSystem = false;
+
+        public static DateTime UtcNow
+        {
+            get
+            {
+                // following code is tuned for speed. Don't change it without running benchmark.
```

Comment:
This comment and the structure of the code does not really make sense - I guess it is a copy&paste from somewhere.

I would change it to:

`return new DateTime(((ulong)(GetSystemTimeAsFileTime() + FileTimeOffset)) | KindUtc);`

Reply:
Yes this is copy and paste from DateTime.CoreClr.cs. but I'll change it as you suggested